src/packages/client/scripts/postinstall.js

```diff
@@ -22,7 +29,18 @@ async function main() {
 
   // this is needed, so that the Generate command does not fail in postinstall
   process.env.PRISMA_GENERATE_IN_POSTINSTALL = 'true'
-
+  if (localPath) {
+    // localPath = xxxx/custom-server/node_modules/prisma/build/index.js
+    // packageRoot = xxxx/custom-server/
+    const packageRoot = localPath && path.join(localPath, '../../../../')
```

Comment:
this should always be the value "true"?

localPath if checked just before in the if

path.join(localPath, '../../../../') is noop, returns a truthy string.

so `PRISMA_GENERATE_IN_POSTINSTALL = packageRoot` means = true? Is that intended like this?

Reply:
So I have made some changes to this logic. I just realized that if people are still using `@prisma/cli` this would not resolve the correct path. Might be worth using some logic from `pkg-up` here.

Reply:
I will implement that when I get in tomorrow morning

Reply:
Ok 💯